tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -743,7 +742,7 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
       result.getEncryptedDeks.isEmpty shouldBe false
 
       val maybeDecrypt =
-        Try(sdk.advanced.documentDecryptUnmanaged(result.getEncryptedData, result.getEncryptedDeks)).toEither
+        Try(sdk.advancedDocumentDecryptUnmanaged(result.getEncryptedData, result.getEncryptedDeks)).toEither
```

Comment:
I think the "advanced" prefix and the "Unmanaged" suffix are very weird together. I know I was the one that suggested the prefix, but after seeing this I think it's probably better just to omit it everywhere.

Reply:
It seems to me that it still serves the purpose of not contaminating the code completion of `sdk.decrypt`.

Reply:
That's true. The "Unmanged" suffix also makes it so that code completion would be reasonable. `sdk.decrypt`<CTL-SPACE> will complete to normal decrypt and then you can also select "Unmanaged" if you want to. I don't feel that strongly about it though, but I do think if we can make this a little simpler it would be good.

Reply:
I'm fine with trying to come up with something later. For this release, I'm going to go with this.